scrapers/codechef_selenium.py

```diff
@@ -0,0 +1,78 @@
+from selenium import webdriver
+from time import sleep
+
+chromeOptions = webdriver.ChromeOptions()
+prefs = {'profile.managed_default_content_settings.images': 2,  # does not load images on web page
+         'disk-cache-size': 1024}  # use disk cache to reduce page load time
+chromeOptions.add_experimental_option("prefs", prefs)
+driver = webdriver.Chrome(options=chromeOptions)
+
+scoreboard_base_url:str = "https://www.codechef.com/rankings"
+site_url="https://www.codechef.com"
+scoreboard_filter_query:str = "?filterBy=Institution%3DPES%20University%2C%20Bengaluru&order=asc&sortBy=rank"
+contest_code:str="APRIL19"
+name_class:str="user-name"
+
+divisions:list=["A","B"]
+problems:dict={'A':set(),'B':set()}
+scraped_scoreboard:dict={'A':list(),'B':list()}
```

Comment:
There seem to be redundant objects with "A" and "B" as attributes. Is it possible to structure it? Something like `namedtuple` would be convenient.